internal/inventory/app/app.go

```diff
@@ -134,6 +144,64 @@ func (app *InventoryApp) Initialize() error {
 	return app.addRoutes()
 }
 
+// bootstrapAliasConfig loads the aliases from the user's configuration toml and pushes them
+// to the config provider if and only if the Aliases key is not present, or the overwrite
+// config flag is passed via the command line
+// todo: switch to using SDK's custom config capability when upgrade to Ireland
+func (app *InventoryApp) bootstrapAliasConfig(sdkFlags flags.Common) error {
+	overwrite := sdkFlags.OverwriteConfig()
+	app.lc.Debug(fmt.Sprintf("Bootstrapping %s config. OverwriteConfig: %v", aliasesConfigKey, overwrite))
```

Comment:
Use `Info` for one time start-up log messages. Same for other Debug messages in this PR that only occur once on start-up.